_platforms/it/io.md

```diff
@@ -16,17 +16,17 @@ github_team: io
 ## Intro
 
 io è l’applicazione del Governo italiano che permetterà a tutti i cittadini di usare servizi pubblici nazionali e locali dal proprio smartphone in maniera semplice, moderna e sicura.
-È un progetto Open Source che crescerà di giorno in giorno, coinvolgendo enti e servizi diversi.
+È un progetto Open Source che cresce di giorno in giorno, coinvolgendo enti e servizi diversi.
```

Comment:
Why the sentence was in the future? The project is growing everyday right now :)

Reply:
Hey @lmillucci, thanks for your work! Can you mention the changes to the Italian part as well in the commit log please?

Reply:
@bfabio done :+1: